src/httpserver.cpp

```diff
@@ -449,6 +449,7 @@ bool UpdateHTTPServerLogging(bool enable) {
 
 std::thread threadHTTP;
 std::future<bool> threadResult;
+static std::vector<std::thread> threadHTTPWorkers;
```

Comment:
Could follow g_thread_http_workers naming convention.

Reply:
Can we please first make sure that this fixes the issue before jumping on naming issues